topics/vanilla-js/index.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases: vanilla-javascript, vanillajs, js-vanilla
```

Comment:
`js-vanilla` is only attached to 3 public repositories. Thoughts on removing that as an alias?
```suggestion
aliases: vanilla-javascript, vanillajs
```

Reply:
Good. Point